### PR TITLE
Rewrites avatars using es6

### PR DESCRIPTION
**Pull request by itsthatguy**
@bigtiger @rylnd @amy

### CODE REVIEW

**Comment by bigtiger** [comment]
@itsthatguy A couple environment-based hiccups otherwise it's looking good.

**Comment by itsthatguy** [comment]
@bigtiger can you elaborate?

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: coffee app.coffee
+web: npm run start:prod
```
Comment (itsthatguy) [inline Procfile]: `npm start` should work... if it doesn't `rm -rf node_modules/` and `npm install`

Comment (bigtiger) [inline Procfile]: This was probably a typo. Works for me now.

**Comment by tjmckenzie** [comment]
@itsthatguy Thank you for this. Comes at a perfect time for me.

**Comment by bigtiger** [comment]
@tjmckenzie Are you running your own instance? How are you using Avatars?

**Comment by tjmckenzie** [comment]
@bigtiger I am in the process of it. I am somewhat new to running my own node server. Currently in the process of it.

I am using these as a form of anonymous chat. I made a slight modification to allow transparent backgrounds, as it would help with my front end design.

**Comment by bigtiger** [comment]
Cool. We deploy to heroku which worked well for a few years until our traffic took off.

**Comment by tjmckenzie** [comment]
Hope too much of that wasn't me (nothing in production yet). I'm now storing them to S3 to limit API requests, and soon running my own instance.

**Comment by bigtiger** [comment]
Haha! Definitely not. There's a few _**very**_ heavy users and a lot of folks using it one off. But, of late we've been serving nearly 30,000,000 per month.

**Pull request merged**